org.tenacityaudio.Tenacity.yaml

```diff
@@ -0,0 +1,222 @@
+      cflags: >
+        -DSQLITE_ENABLE_COLUMN_METADATA=1
+        -DSQLITE_SECURE_DELETE=1
+        -DSQLITE_ENABLE_FTS3=1
+        -DSQLITE_ENABLE_UNLOCK_NOTIFY=1
+        -DSQLITE_ENABLE_DBPAGE_VTAB=1
+        -DSQLITE_ENABLE_DBSTAT_VTAB=1
+    config-opts:
+      - --enable-fts5
+    sources:
+      - type: archive
+        url: https://www.sqlite.org/2020/sqlite-autoconf-3340000.tar.gz
+        sha256: bf6db7fae37d51754737747aaaf413b4d6b3b5fbacd52bdb2d0d6e5b2edd9aee
+
+  - name: chrpath
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://alioth-archive.debian.org/releases/chrpath/chrpath/0.16/chrpath-0.16.tar.gz
+        sha256: bb0d4c54bac2990e1bdf8132f2c9477ae752859d523e141e72b3b11a12c26e7b
+
+  - shared-modules/libmad/libmad.json
+  - shared-modules/linux-audio/jack2.json
+  - shared-modules/linux-audio/lv2.json
+  - shared-modules/linux-audio/lilv.json
+  - deps/suil.json
+
+  - name: rapidjson
+    buildsystem: cmake-ninja
+    config-opts:
+    - -DRAPIDJSON_BUILD_DOC=OFF
+    - -DRAPIDJSON_BUILD_EXAMPLES=OFF
+    - -DRAPIDJSON_BUILD_TESTS=OFF
+    - -DRAPIDJSON_BUILD_THIRDPARTY_GTEST=OFF
+    sources:
+      - type: archive
+        url: https://github.com/miloyip/rapidjson/archive/v1.1.0.tar.gz
+        sha256: bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e
+
+  - name: portaudio
+    sources:
+      - type: git
+        url: https://github.com/PortAudio/portaudio.git
+        tag: v19.7.0
+    buildsystem: cmake-ninja
+
+  - name: tenacity
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_LIBDIR=/app/lib
+      - -Dgtk=gtk+-3.0
+      - -Dglib=glib-2.0
+      - -DwxWidgets_INCLUDE_DIRS=/app/include/wx-3.1;/app/lib/wx/include/gtk3-unicode-3.1
+      - -DwxWidgets_CXX_FLAGS=-pthread
+      - -DwxWidgets_DEFINITIONS_GENERAL=WXUSINGDLL;__WXGTK__;_FILE_OFFSET_BITS=64
+      - -DwxWidgets_LIBRARIES=-lwx_gtk3u_xrc-3.1;-lwx_gtk3u_html-3.1;-lwx_gtk3u_qa-3.1;-lwx_gtk3u_core-3.1;-lwx_baseu_xml-3.1;-lwx_baseu_net-3.1;-lwx_baseu-3.1
+      - -Dlib_preference=system
+      - -Dobey_system_dependencies=On
+      - -Duse_wxwidgets=system
+      - -Duse_soxr=local
+      - -Duse_portaudio=system
+      - -Duse_portmixer=off
+      - -Duse_vamp=local
+      - -Duse_pa_jack=linked
+      - -Duse_midi=local
+      - -Duse_portsmf=local
+      - -Duse_sbsms=local
+      - -Duse_soundtouch=local
+      - -Duse_twolame=local
+      - -Dhas_networking=no
+    post-install:
+      - chrpath -d /app/bin/tenacity
+      - install -Dm644 ../help/tenacity.appdata.xml -t /app/share/metainfo
+      - install -d /app/extensions/Plugins
+    cleanup:
+      - /share/tenacity/include
+      - /share/audacity/include
+      - /share/pixmaps
+    sources:
+      - type: git
+        url: https://github.com/TheEvilSkeleton/tenacity.git
+        tag: fix_appdata
+      - type: patch
+        path: patches/audacity-suil-fix.patch
```

Comment:
about this one, I wish there was a better fix upstream. tl;dr I had a fix for Audacity 2.x, then had to redo it for 3.0.2 and then it broke again with 3.0.3. And yes the PR was left to rot for a year and completely ignored upstream.

Reply:
Ouch, I'm sorry to hear that. Sometimes a ping can make them aware of it again.

Reply:
I took a look at that patch and there's got to be a better way to handle this than directly hacking a hardcoded path into the C++.

Reply:
The changes in https://github.com/audacity/audacity/pull/590 look reasonable to merge into Tenacity though.

Reply:
they no longer worked in 3.0.3 as it is only in the case of in-tree suil. Which in 3.0.3 is no longer the case.

Reply:
> Ouch, I'm sorry to hear that. Sometimes a ping can make them aware of it again.

With their CLA it's a no go.

Reply:
And just to add: there is an error message displayed in the console when the bug occur. Except Audacity decided to suppress console messages in release. Like putting the dust under the rug.... I have some reversal patches for that too.